src/Microsoft.Xaml.Behaviors/Core/LaunchUriOrFileAction.cs

```diff
@@ -34,8 +34,13 @@ public string Path
         protected override void Invoke(object parameter)
         {
             if (this.AssociatedObject != null && !string.IsNullOrEmpty(this.Path))
-            {
-                Process.Start(this.Path);
+            {
+                var processStartInfo = new ProcessStartInfo(this.Path)
+                {
+                    CreateNoWindow = true,
+                    UseShellExecute = true

```

Comment:
This looks good to me. Confirmed this is the default on NETFX already, and is false on `netcoreapp3.0`, so it's essentially a no-op and enables some netcore compatibility.